src/node/historical_queries_adapter.cpp

```diff
@@ -397,18 +398,19 @@ namespace ccf::historical
     return true;
   }
 
-  ccf::endpoints::EndpointFunction adapter_v3(
-    const HandleHistoricalQuery& f,
+  template <class HQY, class EFN, class CTX, class EXT>
```

Comment:
Perhaps add a small comment indicating what these template abbreviations are? Something like: `EFN = EndpointFunction, CTX=EndpointContext, EXT=Extractor` ? (Not sure what `HQY` is)

Reply:
Better yet, name the template parameters as `EndpointFunctionT`, `EndpointContextT`, `ExtractorT` etc ending them with letter `T` to indicate they are template parameters but they also the names give context as to what they represent.

Reply:
Better yet, name the template parameters as EndpointFunctionT, EndpointContextT, ExtractorT etc ending them with letter T to indicate they are template parameters but they also the names give context as to what they represent.

Reply:
Good suggestion - I've renamed these template parameters for clarity.